app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_comments_counter(post_id)
+    post = Post.find_by(id: post_id)
+    post.comments_counter = post.comments.count
+    post.save
+  end
```

Comment:
- [OPTIONAL] Suggestion
```suggestion
  def update_comments_counter
    author.increment!(:posts_counter)
  end
```

**Hint**: *It would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) -* `after_save` to call the `#update_post_counter` method every time a new post has been saved to database, then make the `update_post_counter` method private

```ruby
# `app/models/comment.rb`
  after_save :update_comments_counter

  private

  def update_comments_counter
    author.increment!(:posts_counter)
  end
```